src/main/java/uk/gov/gchq/gaffer/doc/function/ParseDateExample.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.doc.function;
+
+import uk.gov.gchq.koryphe.impl.function.ParseDate;
+
+public class ParseDateExample extends FunctionExample {
+    public static void main(final String[] args) {
+        new ParseDateExample().runAndPrint();
+    }
+
+    public ParseDateExample() {
+        super(uk.gov.gchq.koryphe.impl.function.ParseDate.class);
+    }
+
+    @Override
+    protected void runExamples() {
+        parseDateGMT4();
+        parseDateGMT0();
+    }
+
+    private void parseDateGMT4() {
```

Comment:
There may not be anything you can do but this title gets rendered to "Parse date g m t 4". It would be good if this read something like "Parse date four hours ahead of GMT"